completions/ssh

```diff
@@ -44,8 +44,8 @@ _ssh_options()
         IdentityFile IdentitiesOnly IPQoS KbdInteractiveDevices KexAlgorithms
         LocalCommand LocalForward LogLevel MACs
         NoHostAuthenticationForLocalhost NumberOfPasswordPrompts
-        PasswordAuthentication PermitLocalCommand PKCS11Provider Port
-        PreferredAuthentications Protocol ProxyCommand PubkeyAuthentication
+        PasswordAuthentication PermitLocalCommand PKCS11Provider ProxyJump Port
```

Comment:
Remove this, one addition is enough :)

Reply:
I fixed the issues as mentioned inline and was preparing the -J option when I noticed that ProxyJump accepts comma-separated lists. Since _ssh_suboptions is not supposed to accept these arguments I am going to move the addition to my local config and not push the update.